tools/ecfr-parser/eregs/eregs.go

```diff
@@ -70,3 +78,55 @@ func PostPart(ctx context.Context, p *Part) error {
 	log.Trace("[eregs] Posted ", length, " bytes for part ", p.Name, " version ", p.Date, " in ", time.Since(start))
 	return nil
 }
+
```

Comment:
Noticing some strange spacing errors in this file, on lines 85-101, 104-122, 130-131

Reply:
I think I improved it.  If you have any suggestions, let's sync up and talk them out.